models/experimental/functional_stable_diffusion/tt2/ttnn_functional_resnetblock2d.py

```diff
@@ -456,14 +454,7 @@ def __call__(
                 memory_config=self.second_gn_expected_input_sharded_memory_config,
                 core_grid=self.second_group_norm_core_grid,
             )
-        hidden_states = ttnn.to_memory_config(hidden_states, ttnn.L1_MEMORY_CONFIG)
-        hidden_states = ttnn.reshape(
```

Comment:
What's the input shape here? Don't we need this reshape?

Reply:
I double-checked it with running the test again, it's (1, 1, bhw, c)